src/presentation/service/cron.go

```diff
@@ -0,0 +1,146 @@
+package service
+
+import (
+	"github.com/speedianet/os/src/domain/dto"
+	"github.com/speedianet/os/src/domain/useCase"
+	"github.com/speedianet/os/src/domain/valueObject"
+	cronInfra "github.com/speedianet/os/src/infra/cron"
+	serviceHelper "github.com/speedianet/os/src/presentation/service/helper"
+)
+
+type CronService struct {
+}
+
+func NewCronService() *CronService {
+	return &CronService{}
+}
+
+func (service *CronService) Read() ServiceOutput {
+	cronQueryRepo := cronInfra.CronQueryRepo{}
+	cronsList, err := useCase.GetCrons(cronQueryRepo)
+	if err != nil {
+		return NewServiceOutput(InfraError, err.Error())
+	}
+
+	return NewServiceOutput(Success, cronsList)
+}
+
+func (service *CronService) Create(input map[string]interface{}) ServiceOutput {
+	requiredParams := []string{"schedule", "command"}
+	err := serviceHelper.RequiredParamsInspector(input, requiredParams)
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	cronSchedule, err := valueObject.NewCronSchedule(input["schedule"])
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	cronCommand, err := valueObject.NewUnixCommand(input["command"])
+	if err != nil {
+		return NewServiceOutput(UserError, err.Error())
+	}
+
+	var cronCommentPtr *valueObject.CronComment
+	if input["comment"] != nil {
+		cronComment, err := valueObject.NewCronComment(input["comment"])
+		if err != nil {
+			return NewServiceOutput(UserError, err.Error())
+		}
+		cronCommentPtr = &cronComment
+	}
+
+	dto := dto.NewCreateCron(cronSchedule, cronCommand, cronCommentPtr)
+
+	cronCmdRepo, err := cronInfra.NewCronCmdRepo()
+	if err != nil {
+		return NewServiceOutput(InfraError, err.Error())
+	}
+
+	err = useCase.CreateCron(cronCmdRepo, dto)
+	if err != nil {
+		return NewServiceOutput(InfraError, err.Error())
+	}
+
+	return NewServiceOutput(Success, "CronCreated")
```

Comment:
Created, not Success. Success will become 200 OK on HTTP while Created will be 201 Created.

Reply:
As discussed, we will keep `cmdRepo` being initialized in all methods. However, `queryRepo` has been moved to the constructor.

Reply:
Done.